Placeholders.md

```diff
@@ -3386,6 +3387,72 @@ Please see [this discussion][list] for a list of all expansions officially maint
   
 ----
 
+- ### **[LotterySix](https://www.spigotmc.org/resources/107573/)**
+  > NO DOWNLOAD COMMAND
+  ```
```

Comment:
You should add an empty line between the quote and the code block for readability.